src/Resources/config/config.xml

```diff
@@ -117,4 +117,24 @@
             </helpText>
         </input-field>
     </card>
+    <card>
+        <title>Delayed Capture</title>
+        <input-field type="bool">
+            <name>active</name>
+            <label>Active</label>
+        </input-field>
+        <input-field type="int">
+            <name>rescheduleTime</name>
+            <label>Reschedule Notification Time</label>
```

Comment:
We might reconsider this configuration field in a follow-up PR, as it doesn't sound too user friendly.